### ISSUE TITLE
[Feature] Upload file from client

### ISSUE THREAD

**Issue by HijackHornet** [stashapp/stash#2558]
**Problem.**
Lets say i'm hosting my stash on a distant machine (as linux server for instance) and i navigate the stash from the web browser on an Android device. It would be neat to be able to upload a file (picture, gif or small video) to my stash from my android device.

**Solution**
An upload button in the navbar that allow me to select one or multiple files to upload. Once the upload is complete, the database will refresh automatically

**The current solution to doing this**
At the moment, the user would have to use an android FTP client, setup and FTP server on the distant server, and then go into the client UI to trigger a rescan. This is very cumbersome

**Additional context**
I've heard this feature wouldnt be useful to all as most people use stash on the same device than their content. But it would make is much more versatile.

**Comment by WithoutPants** [comment]
I'm going to say this is a duplicate of #13 - even though it only says scene.
